vinyldns/batch_changes_resources.go

```diff
@@ -40,9 +40,21 @@ type RecordChange struct {
 
 // BatchRecordChangeUpdateResponse is represents a batch record change create or update response
 type BatchRecordChangeUpdateResponse struct {
-	Comments     string         `json:"comments,omitempty"`
-	OwnerGroupID string         `json:"ownerGroupId,omitempty"`
-	Changes      []RecordChange `json:"changes,omitempty"`
+        ID                 string         `json:"id,omitempty"`
```

Comment:
so this looks like it has extra indentation here, but when i view the file it shows the ID attribute aligned with everything else.